action.yml

```diff
@@ -1,10 +1,17 @@
-name: 'Node 12 Template Action'
-description: 'Get started with Node actions'
-author: 'GitHub'
+name: 'Setup Alire'
+description: 'Prepare Alire to be run in subsequent actions'
+author: 'Alire Project'
+
 inputs:
-  myInput:
-    description: 'Input to use'
-    default: 'world'
+  toolchain:
+    description: Arguments to pass to `alr toolchain` after setup
+    required: false
+    default: 'gnat_native gprbuild<2000'
```

Comment:
Why `<2000` for gprbuild?

Reply:
Otherwise it picks the gprbuild from Ubuntu/Debian, which is 2019/2020, and that one does not recognize our `gnat_native`. 

An alternative would be to have separate crates such as `gprbuild_external` and `gprbuild`, or `gprbuild_source`, but right now there is nothing preventing mixing external/packaged gnat/gprbuild, which don't recognize each other. May be we should hardcode something about it, or make our GNATs depend explicitly on one of the packaged `gprbuild`. I have to consider this more in depth, but this should serve as a temporary workaround.

Reply:
> Otherwise it picks the gprbuild from Ubuntu/Debian, which is 2019/2020,

:roll_eyes: :frowning_face: 

> An alternative would be to have separate crates such as `gprbuild_external` and `gprbuild`, or `gprbuild_source`, but right now there is nothing preventing mixing external/packaged gnat/gprbuild, which don't recognize each other.

Having a `gprbuild_external` like we have a `gnat_external` seems the way to go.

Reply:
That alone is not enough, because if both `gprbuild_external` and `gprbuild` (from sources) provide the same `gprbuild` crate, there is no way to specify that you want a non-external gprbuild. And `gprbuild_native` doesn't really make sense in this case. All in all I'm not sure how to do it neatly and non-confusingly so to say.

Reply:
This is so annoying...

Reply:
Maybe a little bit of magic in `alr toolchain` is the simplest solution. Something that ensures that these mixes can't occur, and limits the available `gprbuild` depending on the selected gnat.